pyproject.toml

```diff
@@ -4,50 +4,55 @@
 
 [project]
 name = 'easydiffraction'
-dynamic = ['version']  # Use versioningit to manage the version
+dynamic = ['version'] # Use versioningit to manage the version
 description = 'Making diffraction data analysis and modelling easy'
 authors = [
-    { name = 'EasyDiffraction contributors', email = 'support@easydiffraction.org' }
+  { name = 'EasyDiffraction contributors', email = 'support@easydiffraction.org' },
 ]
 readme = 'README.md'
 license = { file = 'LICENSE' }
 classifiers = [
-    'Topic :: Scientific/Engineering',
-    'Development Status :: 4 - Beta',
-    'License :: OSI Approved :: BSD License',
-    'Operating System :: OS Independent',
-    'Programming Language :: Python :: 3 :: Only',
-    'Programming Language :: Python :: 3',
-    'Programming Language :: Python :: 3.9',
-    'Programming Language :: Python :: 3.10',
-    'Programming Language :: Python :: 3.11',
-    'Programming Language :: Python :: 3.12'
+  'Topic :: Scientific/Engineering',
+  'Development Status :: 4 - Beta',
+  'License :: OSI Approved :: BSD License',
+  'Operating System :: OS Independent',
+  'Programming Language :: Python :: 3 :: Only',
+  'Programming Language :: Python :: 3',
+  'Programming Language :: Python :: 3.9',
+  'Programming Language :: Python :: 3.10',
+  'Programming Language :: Python :: 3.11',
+  'Programming Language :: Python :: 3.12',
 ]
 requires-python = '>=3.9,<3.13'
 dependencies = [
-    'cryspy',  # For calculations of diffraction patterns
-    'pooch',  # For data fetching
-    'easyscience',  # The base library of the EasyScience framework
-    'easycrystallography',  # Crystallography tools of the EasyScience framework
+  'cryspy',              # Calculations of diffraction patterns
+  'easycrystallography', # Crystallography tools of the EasyScience framework
+  'easyscience',         # The base library of the EasyScience framework
+  'pooch',               # For data fetching
```

Comment:
I thought you argued to keep comments aligned with code not with each other, but this solution here does look nicer.

Reply:
This formatting is now done automatically by Prettier, so there is no need to worry about aligning comments yourself 😉